pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <version.eclipse.microprofile.config>1.4</version.eclipse.microprofile.config>
-        <version.eclipse.microprofile.graphql>1.0.1</version.eclipse.microprofile.graphql>
+        <version.eclipse.microprofile.graphql>1.0.2</version.eclipse.microprofile.graphql>
```

Comment:
We'll have to wait for 1.0.2 release before we can merge this - but hopefully that will be very soon!